biochemist-python/chapters/Working_with_Pandas.ipynb

```diff
@@ -0,0 +1,1172 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Working with Pandas\n",
+    "================="
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"overview-this-is-a-title overview\">\n",
+    "<h2 class=\"overview-title\">Overview</h2>\n",
+    "    \n",
+    "<p>Questions</p>\n",
+    "    <ul>\n",
+    "        <li>How can I use import data for analysis in my notebook?\n",
+    "    </ul>\n",
+    "<p>Objectives:</p>\n",
+    "    <ul>\n",
+    "        <li>Import the pandas library.\n",
+    "        <li>Use pandas library funtions to import data from a csv file.\n",
+    "        <li>Import data from a .csv formatted file.\n",
+    "    </ul>\n",
+    "<p>Keypoints:</p>\n",
+    "    <ul>\n",
+    "        <li>Use the pandas library to create dataframes from csv formatted data.</li>\n",
+    "    </ul>\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## What is pandas and why do we use it?\n",
+    "\n",
+    "Pandas is a python library that is designed to work with two dimensional data arrays. It is built on numpy, another python library that specializes in numerical analysis. Numpy also has the ability to create and analyze data arrays. If you are not familiar with arrays, here are some simple examples.\n",
+    "\n",
+    "### 1D arrays\n",
+    "A one dimensional array is simply a list of items, for example, a list of the elements: H, He, Li, Be, B, etc.\n",
+    "\n",
+    "### 2D arrays\n",
+    "A two dimensional array contains matched pairs of information. A list of elements with their atomic numbers would be a 2D array:  (H, 1), (He, 2), (Li, 3), (Be, 4), (B, 5), etc. It could also be a list of x, y data points that you want to plot: (1,1), (2,3), (3,5), (4,7). \n",
+    "\n",
+    "### 3D arrays\n",
+    "A three dimensional array contains matched triads of information. One common 3D array in biochemistry and structural biology would be a collection of x, y, and z coordinates for the atoms in a protein, as are found in the [Protein Data Bank](http://www.rcsb.org/pdb). \n",
```

Comment:
This is still a two dimensional array. A three-dimensional array would be a collection of two dimensional arrays. This might be a collection of coordinates through time, for example.  

A two dimensional array has rows and columns, and three dimensional array has row, columns and a third dimension. See the figure for 3 dimensional arrays at this site - https://owlcation.com/stem/How-to-work-with-Multidimensional-Array-in-C-Programming

Reply:
I changed the text to read, "A three-dimensional array would be a collection of two dimensional arrays. For example, this might be a collection of x, y, and z coordinates for a structure as a function of time."

3D arrays are not really important to this lesson series, so I don't think we need to say any more about them.